src/averageAges.js

```diff
@@ -38,6 +53,21 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const women = people.filter(person => person.sex === 'f');
+  let womenChild = women;
+
+  if (withChildren) {
+    womenChild = women.filter(person => {
+      return people.find(wom => person.name === wom.mother);
```

Comment:
I don't know how to apply the filter here. Here I am looking for women with at least one child, if you replace "find" with "filter" then there will be duplicate mothers (18 mothers instead of 13 unique ones). Is it possible to see the mentor's decision?